README.md

```diff
@@ -163,18 +168,20 @@ implementation 'com.svydovets:svydovets-bring-framework:0.0.1-SNAPSHOT'
     <server>
         <id>github</id>
         <username>your_github_email</username>
-        <password>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</password>
+        <password>bring_token</password>
     </server>
 </servers>
 </settings>             
 ```
-3. Replace your_github_email in this file on your email from the GitHub account
-4. Add dependency:
+3. Replace *your_github_email* in this file on your email from the GitHub account
+4. [Contact Bring team and ask a token for downloading dependency](https://github.com/khshanovskyi/get_token/blob/main/README.md)
+5. Replace *bring_token* on provided from Bring team
+6. Add dependency:
 ```
 <dependency>
-       <groupId>com.svydovets</groupId>
-       <artifactId>svydovets-bring-framework</artifactId>
-       <version>0.0.1-SNAPSHOT</version>
+     <groupId>com.svydovets.documented</groupId>
```

Comment:
I thought to push the standard jar after todays PR. So, the final groupId will be `com.svydovets`